app/Kconfig

```diff
@@ -424,6 +424,10 @@ choice CBPRINTF_IMPLEMENTATION
 
 endchoice
 
+config ZMK_USB_REPORT_LEDS
```

Comment:
Maybe it would make more sense to just always support reading the report? Unless there's some reason that the HID report descriptor might need to be kept short, I think making this conditional would just make things more confusing. Any features that depend on this would also need to be conditional, and then anyone who wants to use it in their code has to know to turn it on.